oshdb-filter/src/main/java/org/heigit/ohsome/oshdb/filter/FilterExpression.java

```diff
@@ -40,6 +43,7 @@
    */
   @Contract(pure = true)
   default boolean applyOSH(OSHEntity entity) {
+    // (potentially slow) default implementation tests every version individually
     return Streams.stream(entity.getVersions()).anyMatch(this::applyOSM);
```

Comment:
What do you think about to just return true here in case we don't have a OSHEntity filter implementation.
As far as I know when an OSH pass the OSHFilter its versions OSM will be filter as well. So is there any benefit to do it twice?

```suggestion
    return true;
```

Reply:
I think you are right. The celliterator can/will even skip testing of some of the versions for example, if they are outside of the query's time interval. :+1: